commerce/endpoint/cart/CartItemCollectionExtensionSample.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>62.0</apiVersion>
```

Comment:
We created the Apex class using the Salesforce CLI command. This auto-generated file contained the API version 59.0, which we updated to 62.0.